sys/shell/commands/sc_gnrc_6lo_frag_stats.c

```diff
@@ -28,6 +28,8 @@ int _gnrc_6lo_frag_stats(int argc, char **argv)
 #ifdef MODULE_GNRC_SIXLOWPAN_FRAG_VRB
     printf("VRB full: %u\n", stats->vrb_full);
 #endif
+    printf("frags complete: %u\n", stats->fragments);
+    printf("dgs complete: %u\n", stats->datagrams);
```

Comment:
```suggestion
    printf("dgs complete: %u\n", stats->datagrams);
#endif
```

Reply:
Add a dangling `#endif`? o.O

Reply:
Aaaaaaahhhh supposed to be paired with the `#ifdef`s you proposed!